Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PipelineBrush/PipelineBrushXaml.bind

```diff
@@ -0,0 +1,46 @@
+﻿<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:brushes="using:Microsoft.Toolkit.Uwp.UI.Media.Brushes"
+    xmlns:effects="using:Microsoft.Toolkit.Uwp.UI.Media.Effects"
+    mc:Ignorable="d">
+
+  <Grid>
+    <Image Source="ms-appx:///Assets/Photos/BigFourSummerHeat.jpg"/>
+    <Grid>
+      <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="*" />
+        <ColumnDefinition Width="*" />
+      </Grid.ColumnDefinitions>
+      <ProgressRing IsActive="True" Grid.ColumnSpan="2"
+                    VerticalAlignment="Center" HorizontalAlignment="Center" Foreground="Red" Width="200" Height="200"/>
+      <Border BorderBrush="Black" BorderThickness="1"
+              Grid.Column="2"
+              Height="400">
+        <Border.Background>
+          <brushes:PipelineBrush>
+            <brushes:PipelineBrush.Effects>
+              <effects:BackdropEffect Source="Backdrop"/>
+              <effects:LuminanceToAlphaEffect/>
+              <effects:OpacityEffect Value="0.4"/>
+              <effects:BlendEffect Mode="Multiply">
+                <effects:BlendEffect.Input>
+                  <effects:BackdropEffect Source="Backdrop"/>
```

Comment:
Could be interesting to see an image like the unicorn one we have in the sample app composited via the brush?

Reply:
Consider it done! 🚀

![image](https://user-images.githubusercontent.com/10199417/82105840-c6994280-971d-11ea-87ac-6cbdf084ba3e.png)